specification/maps/data-plane/Weather/preview/1.0/weather.json

```diff
@@ -555,6 +698,203 @@
+          },
+          {
+            "$ref": "#/parameters/OptionalWeatherGovernmentId"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/StormSearchResponse"
+            }
+          },
+          "default": {
+            "$ref": "../../../Common/preview/1.0/common.json#/responses/default"
+          }
+        }
+      }
+    },
+    "/weather/tropical/storms/forecasts/{format}": {
+      "get": {
+        "description": "**Get Storm Forecasts**\n\n**Applies to**: Gen 1 (S0 and S1) and Gen 2 pricing tiers.\n\nGet all government-issued tropical storm forecasts. Information about the forecasted tropical storms includes, location, status, date the forecast was created, window, wind speed and wind radii.",
+        "operationId": "Weather_GetStormForecast",
+        "x-ms-examples": {
+          "Get Storm Forecasts": {
+            "$ref": "./examples/GetStormForecast.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "../../../Common/preview/1.0/common.json#/parameters/ClientId"
+          },
+          {
+            "$ref": "#/parameters/ApiVersion"
+          },
+          {
+            "$ref": "../../../Common/preview/1.0/common.json#/parameters/JsonFormat"
+          },
+          {
+            "$ref": "#/parameters/WeatherYear"
+          },
+          {
+            "$ref": "#/parameters/RequiredWeatherBasinId"
+          },
+          {
+            "$ref": "#/parameters/RequiredWeatherGovernmentId"
+          },
+          {
+            "$ref": "#/parameters/WeatherDataUnit"
+          },
+          {
+            "$ref": "#/parameters/WeatherDetails"
+          },
+          {
+            "$ref": "#/parameters/WeatherRadiiGeometry"
+          },
+          {
+            "name": "windowGeometry",
+            "description": "When true, window geometry data (geoJSON) is included in the response",
+            "in": "query",
+            "type": "boolean",
+            "default": false
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/StormForecastResponse"
+            }
+          },
+          "default": {
+            "$ref": "../../../Common/preview/1.0/common.json#/responses/default"
+          }
+        }
+      }
+    },
+    "/weather/tropical/storms/locations/{format}": {
+      "get": {
+        "description": "**Get Storm Locations**\n\n**Applies to**: Gen 1 (S0 and S1) and Gen 2 pricing tiers.\n\nGet locations of an individual government-issued tropical storms. Information about the tropical storms includes, location coordinates, geometry, basin ID, date, wind details and wind radii.",
+        "operationId": "Weather_GetStormLocations",
+        "x-ms-examples": {
+          "Get Storm Locations": {
```

Comment:
Can we change this to 'Get Tropical Storm Locations'? I think the name should be more explicit, especially when it's listed with all the other Weather Services once in production, we want users to know it's tropical storms not regular storms.

Reply:
Updated